hack/verify-dockerless.sh

```diff
@@ -0,0 +1,15 @@
+not_have_dockerless=$(find ../pkg/kubelet/dockershim -name "*.go" ! -exec grep -q 'dockerless' {} \; -print)
```

Comment:
If you look at the scripts in `hack/` you can see that they all share some common boilerplate (specifically lines 1-23 in this [file](https://sourcegraph.com/github.com/kubernetes/kubernetes@master/-/blob/hack/verify-typecheck-dockerless.sh)). Please add that boilerplate here.

Reply:
Also, once you add the boilerplate, the script will be running the KUBE_ROOT directory so you will want to update all paths from `../pkg/kubelet` to `./pkg/kubelet`.